app.rb

```diff
@@ -14,25 +14,6 @@ def initialize
     @people = []
   end
 
```

Comment:
- [ ] In our `app.rb` file, to follow the Single Responsiblity Rule, YAGNI, KISS and SOLID [lessions](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/lessons/solid.md#best-practices-and-solid), all request for input and input validations should be placed in the `main.rb` file. For example, lets us take one of the methods `create_teacher` in line `69-74` all these are input and input validation that should be moved into the `main.rb` file. **NB**: I only used this method as an example, kindly use this same principle in all

Reply:
Thank you for those changes.